plugins/catalog/src/components/CatalogFilter/CatalogFilter.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Card,
+  List,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Typography,
+  Theme,
+  makeStyles,
+} from '@material-ui/core';
+import { IconComponent } from '@backstage/shared/icons';
```

Comment:
This doesn't exist does it? 😅

Reply:
Nope it's why the build failed. I only just realised how the packages are setup that you have to export them from `index` in `@backstage/core`